build/images/scripts/start_ovs

```diff
@@ -64,7 +64,13 @@ trap "quit" INT TERM
 
 cleanup_ovs_run_files
 
-start_ovs
+offload=false
+if [ $# == 1 ] && [ -n "$1" ] && [ "$1" == "--hw-offload" ]; then
+    log_info $CONTAINER_NAME "Set OVS to hw-offload=true"
+    offload=true
+fi
+
+start_ovs $offload
```

Comment:
I did not see you change start_ovs func?

Reply:
sorry my bad I will update the PR